docker/gen-run-image

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+set -e
+
+PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
+ROOT=$(git rev-parse --show-toplevel)
+
+export DIST="xenial"
+export DMD="dmd1"
+export F="production"
+
+cd $ROOT
+BEAVER_DOCKER_Iimg="sociomantictsunami/dhtnode-builder" beaver dlang install
```

Comment:
https://github.com/sociomantic-tsunami/beaver/pull/39

Reply:
Fixed and v0.3.2 released.